libsolidity/formal/SMTEncoder.cpp

```diff
@@ -738,6 +742,32 @@ void SMTEncoder::visitGasLeft(FunctionCall const& _funCall)
 		m_context.addAssertion(symbolicVar->currentValue() <= symbolicVar->valueAtIndex(index - 1));
 }
 
+void SMTEncoder::visitAddMod(FunctionCall const& _funCall)
```

Comment:
Can `visitAddMod` and `visitMulMod` be one function? Code looks the same except last line which differs only by applying `+` or `*` depending on which operation we are in.

Reply:
Yea I'll merge them